common/src/main/kotlin/entity/Permission.kt

```diff
@@ -327,6 +338,7 @@ public sealed class Permission(public val code: DiscordBitSet) {
                 ManageRoles,
                 ManageWebhooks,
                 ManageEmojisAndStickers,
```

Comment:
this should be replaced by the renamed one

Reply:
fixed